### PR TITLE
S3 tutorial floc 1718

### PR DESCRIPTION
**Pull request by adamtheturtle**
This should be merged at the same time as https://github.com/ClusterHQ/build.clusterhq.com/pull/74.

### CODE REVIEW

File: docs/gettinginvolved/infrastructure/release-process.rst [hunk 1]
```diff
@@ -351,7 +353,7 @@ Release
 
    .. prompt:: bash [vagrant@localhost]$
 
-      echo https://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-${VERSION}.box
+      echo https://s3.amazonaws.com/clusterhq-archive/vagrant/tutorial/flocker-tutorial-${VERSION}.box
```
Comment (adamtheturtle) [inline docs/gettinginvolved/infrastructure/release-process.rst]: Remove the section above, starting "# Copy the tutorial box to the final location"

File: admin/release.py [hunk 2]
```diff
@@ -431,6 +431,24 @@ def update_repo(package_directory, target_bucket, target_key, source_repo,
...
+    """
+    Copy the tutorial box from a ``dev_bucket`` to a ``target_bucket``.
+
+    :param bytes target_bucket: S3 bucket to copy tutorial box to.
+    :param bytes dev_bucket: S3 bucket to copy tutorial box from.
+    :param bytes version: Version of Flocker to copy the tutorial box for.
+    """
+    yield Effect(
+        CopyS3Keys(source_bucket=dev_bucket,
+                   source_prefix='vagrant/tutorial/'.format(version),
```
Comment (tomprince) [inline admin/release.py]: `.format(version)` does nothing here.

File: admin/test/test_release.py [hunk 3]
```diff
@@ -1824,3 +1824,38 @@ def test_tag_exists_fails(self):
...
+        """
+        target_bucket = 'clusterhq-archive'
+        dev_bucket = 'clusterhq-dev-archive'
+
+        aws = FakeAWS(
+            routing_rules={},
+            s3_buckets={
+                target_bucket: {},
+                dev_bucket: {
+                    'vagrant/tutorial/flocker-tutorial-0.3.0.box': '',
```
Comment (tomprince) [inline admin/test/test_release.py]: Put some content here, to verify that it gets copied.

**Comment by tomprince** [comment]
A couple of minor comments to address, then merge

**Pull request merged**